boards/arduino/uno_r4/board.yml

```diff
@@ -1,11 +1,9 @@
-boards:
-  - name: arduino_uno_r4_minima
-    full_name: Arduino UNO R4 Minima
-    vendor: arduino
-    socs:
-      - name: r7fa4m1ab3cfm
-  - name: arduino_uno_r4_wifi
-    full_name: Arduino UNO R4 WiFi
-    vendor: arduino
-    socs:
-      - name: r7fa4m1ab3cfm
+board:
+  name: arduino_uno_r4
+  full_name: Arduino UNO R4 Minima/WiFi
+  vendor: arduino
+  socs:
+    - name: r7fa4m1ab3cfm
+      variants:
+        - name: minima
+        - name: wifi
```

Comment:
these are 2 different boards, should be a board revision not a variant. Variant is same hardware but different configuration (or a manual board change like jumper change/solder pad change)